.github/workflows/publish.yml

```
@@ -17,7 +17,10 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-
+        with:
+          fetch-tags: true
+      - name: Get tags
+        run: git fetch --tags origin
```

```suggestion
        with:
          fetch-depth: 0
```
maybe this is enough?

---

Yeah, sure, it will work, I just figured we could avoid fetching the entire history.

---

@einarmo in that case you use your approach for sure